pkg/vmtest/gotest.go

```diff
@@ -50,6 +50,7 @@ func GolangTest(t *testing.T, pkgs []string, o *Options) {
 	env.CgoEnabled = false
 	env.GOARCH = TestArch()
 	o.BuildOpts.Env = env
+	o.BuildOpts.UrootSource = filepath.Join(env.GOPATH, "src/github.com/u-root/u-root")
```

Comment:
Same, this will break people who run our VM tests on the command line locally if cloned not in GOPATH.